other/concat-exports.js

```diff
@@ -0,0 +1,45 @@
+const appendToFile = require('fs').appendFileSync
+const resolve = require('path').resolve
+const htmlTagNames = require('html-tag-names')
+const svgTagNames = require('svg-tag-names')
+
+function capitalize(s) {
+  return s.slice(0, 1).toUpperCase() + s.slice(1)
+}
+
+function dashToCamelCase(s) {
+  return s.replace(/-([a-z])/g, ([, char]) => char.toUpperCase())
+}
+
+function isCapitalized(s) {
+  return !!s && s[0] === s[0].toUpperCase()
+}
+
+// node version <= 7 does not support trailing commas in function calls
```

Comment:
Normally, I'll add [this](https://github.com/paypal/glamorous/blob/b7424288431878bace8d9bb3b7d641e21a61b78f/package-scripts.js#L111-L123) at the end of files which are not transpiled:

```javascript
// this is not transpiled
/*
  eslint
  max-len: 0,
  comma-dangle: [
    2,
    {
      arrays: 'always-multiline',
      objects: 'always-multiline',
      functions: 'never'
    }
  ]
 */
```